### PR TITLE
List USDC-ETH on the frontend [v2]

### PR DESCRIPTION
**Pull request by vaibhavgeek**
This can now be merged.  https://github.com/Ricochet-Exchange/ricochet-frontend/issues/298

### CODE REVIEW

File: src/api/ethereum.ts [hunk 1]
```diff
@@ -130,33 +129,19 @@ export const startFlow = async (
   });
   let call = [];
 
-  const isSubscribedUSDC = await idaContract.methods
-    .getSubscription(
-      USDCxAddress,
-      twoWayMarketAddress, // publisher
```
Comment (mikeghen) [inline src/api/ethereum.ts:136]: This variable is imported, use`exchangeAddress` which in this context will be the `twoWayMarketAddress`

File: src/api/ethereum.ts [hunk 2]
```diff
@@ -130,33 +129,19 @@ export const startFlow = async (
...
-    .call();
+  // eslint-disable-next-line max-len
+  const config = indexIDA.filter((data) => data.input === inputTokenAddress && data.output === outputTokenAddress)[0];
 
-  const isSubscribedETH = await idaContract.methods
+  const isSubscribed = await idaContract.methods
     .getSubscription(
-      WETHxAddress,
+      config.output,
       twoWayMarketAddress, // publisher
```
Comment (mikeghen) [inline src/api/ethereum.ts:138]: This variable is imported, use `exchangeAddress` which in this context will be the `twoWayMarketAddress`

File: src/api/ethereum.ts [hunk 3]
```diff
@@ -213,7 +180,7 @@ export const startFlow = async (
                 superFluid.agreements.cfa.contract.methods
                   .createFlow(
                     inputTokenAddress,
-                    twoWayMarketAddress,
+                    exchangeAddress,
```
Comment (mikeghen) [inline src/api/ethereum.ts:183]: Here you made the replacement correctly

**Review by mikeghen — CHANGES_REQUESTED** [review]

File: src/api/ethereum.ts [hunk 4]
```diff
@@ -130,33 +129,19 @@ export const startFlow = async (
...
-          address: twoWayMarketAddress,
-          token: inputTokenAddress,
-        }), // address: would be rickosheaAppaddress, currently not deployed
-        flowRate: amount.toString(),
-      });
-    } else if (isSubscribedETH.approved && inputTokenAddress === USDCxAddress) {
+    if (isSubscribed.approved) {
       await sfUser.flow({
         recipient: await superFluid.user({
           address: twoWayMarketAddress,
```
[the remark this answers is not in the document]
Comment (vaibhavgeek) [reply src/api/ethereum.ts]: I have made the necessary changes!

File: src/api/ethereum.ts (hunk 2 above)
Comment (vaibhavgeek) [reply src/api/ethereum.ts]: I have changed it here as well.

**Review by mikeghen — APPROVED** [review]

**Comment by mikeghen** [comment]
Tested this on the link above, only tested starting stopping streams on existing accounts

**Pull request merged**